src/main/resources/css/maven-theme.css

```diff
@@ -160,3 +160,8 @@ h5 > code {
 .table td {
   text-align: revert;
 }
+
+/* linked code must be styled in same color as plain links */
+a > code {
+  color: #0088cc;
```

Comment:
for some reason I couldn't get it to work with the same slightly different blue color for hovered/focused links, but this is already better than before.

Reply:
As long as it is now visually distinguishable, I am fine with it.